src/plugins/intel_cpu/src/edge.h

```diff
@@ -87,7 +87,9 @@ class Edge {
     std::weak_ptr<Node> parent;
     std::weak_ptr<Node> child;
     int parent_port;
-    int child_port;
+    // Changed to handle full range of size_t, matching the value being assigned
+    // to it
+    size_t child_port;
```

Comment:
Couly you please share warning logs that highlight what exact lines in the code have conversion issues? I don't think changing type of hte port is best solution, so would be nice to understand alternatives.